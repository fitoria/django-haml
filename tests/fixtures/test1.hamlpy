#profile
    .left.column
        #date 2010/02/18
        #address Toronto, ON
    .right.column
        #bio Jesse Miller
