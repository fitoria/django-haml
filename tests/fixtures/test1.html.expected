<div id='profile'>
    <div class='left column'>
        <div id='date'>2010/02/18</div>
        <div id='address'>Toronto, ON</div>
    </div>
    <div class='right column'>
        <div id='bio'>Jesse Miller</div>
    </div>
</div>
